node/network/gossip-support/src/lib.rs

```diff
@@ -404,9 +403,7 @@ where
 			NetworkBridgeEvent::OurViewChange(_) => {},
 			NetworkBridgeEvent::PeerViewChange(_, _) => {},
 			NetworkBridgeEvent::NewGossipTopology { .. } => {},
-			NetworkBridgeEvent::PeerMessage(_, Versioned::V1(v)) => {
-				match v {};
-			},
+			NetworkBridgeEvent::PeerMessage(_, _) => {},
```

Comment:
I haven't checked this in a while, but I remember a few years ago it was the case that matching on an empty enum was guaranteed to emit and LLVM `unreachable` instruction which allowed entire codepaths to be optimized away.

It shouldn't practically impact performance to remove that at all.